clang/include/clang/Basic/DiagnosticASTKinds.td

```diff
@@ -410,10 +410,9 @@ def warn_integer_constant_overflow : Warning<
 def warn_fixedpoint_constant_overflow : Warning<
   "overflow in expression; result is %0 with type %1">,
   InGroup<DiagGroup<"fixed-point-overflow">>;
-def warn_constexpr_unscoped_enum_out_of_range : Warning<
+def note_constexpr_unscoped_enum_out_of_range : Note<
   "integer value %0 is outside the valid range of values [%1, %2] for the "
-  "enumeration type %3">, DefaultError, ShowInSystemHeader, ShowInSystemMacro,
-  InGroup<DiagGroup<"enum-constexpr-conversion">>;
```

Comment:
Probably worth noting in the release notes that the diagnostic group also is removed and so users can expect an "unknown warning" diagnostic as well.